spec/javascripts/helpers/SpecHelper.js

```diff
@@ -40,3 +40,7 @@ function isElementVisible(element) {
 function isElementDisabled(selector) {
   return document.querySelector(selector+':disabled') !== null
 }
+
+var isElementHidden = function(element) {
```

Comment:
Why is `isElementHidden` declared differently from `isElementDisabled`? Fine if there's a reason, but otherwise would be good to be consistent (I think the two ways of declaring are basically the same anyway).

Reply:
I reviewed this commit by commit so I think this has already been addressed later in the PR.

Reply:
@andysellick Yeah sorry I had used this declaration and then James had used a different declaration, so it only got standardised after I rebased.